src/internal.rs

```diff
@@ -188,28 +187,34 @@ impl Local {
     const PINNINGS_BETWEEN_COLLECT: usize = 128;
 
     /// Registers a new `Local` in the provided `Global`.
-    pub fn register(global: &Arc<Global>) -> Handle {
+    pub fn register(collector: &Collector) -> Handle {
```

Comment:
This is a really minor nit, but wouldn't it make more sense to accept a collector by value rather than reference here?

Reply:
I changed it because of line 203: we need to have `collector` after making `local`.